arch/arm/core/aarch32/prep_c.c

```diff
@@ -29,6 +29,10 @@
 #include <aarch32/cortex_a_r/stack.h>
 #endif
 
+#if defined(CONFIG_CPU_CORTEX_M)
+#include <aarch32/cortex_m/early_init.h>
+#endif
```

Comment:
why is this `CORTEX_M` specific?

Reply:
I was thinking the same looking at this ... why not A/R also?